### ISSUE TITLE
Demo Fundamentally Broken

### ISSUE THREAD

**Issue by Ceroxylon** [dockersamples/101-tutorial#34]
I copy and pasted the code for creating the Dockerfile.

First issue: yarn has changed its commands, it is now "add" instead of "install"

Second Issue: after giving the 'docker build -t docker-101 .' command, the first three steps are successful, but it gives an error at step 4 after it tries to add v1.22.5, saying

"error Missing list of packages to add to your project."

Not exactly inspiring to learn Docker when the tutorial requires so much debugging

**Comment by gggirlgeek** [comment]
I agree! And how do you create a Dockerfile???? The tutorial should say. I ended up creating a blank text document on my home computer named "Dockerfile" and drag-dropping it onto the PWD command line to upload it. Then I pressed the Edit tab and the file was  there under the app folder. Clicking it opens it for editing.
